apm-agent-plugins/apm-api-plugin/src/main/java/co/elastic/apm/agent/pluginapi/ApiInstrumentation.java

```diff
@@ -18,12 +18,17 @@
  */
 package co.elastic.apm.agent.pluginapi;
 
-import co.elastic.apm.agent.bci.TracerAwareInstrumentation;
+import co.elastic.apm.agent.impl.Tracer;
+import co.elastic.apm.agent.sdk.ElasticApmInstrumentation;
+import co.elastic.apm.agent.tracer.GlobalTracer;
 
 import java.util.Collection;
 import java.util.Collections;
 
-public abstract class ApiInstrumentation extends TracerAwareInstrumentation {
+public abstract class ApiInstrumentation extends ElasticApmInstrumentation {
```

Comment:
I wanted to delay migrating the plugins as the API instrumentations use plenty of "unique" APIs and added therefore a static `tracer` field which unwrapped to the old API. Turns out this was a bad idea anyways as the tracer might not be initialized such that the `require` call would fail and cause the unit tests to fail, too. I moved this back and rather require the tracer implementation where needed, inline.